lib/src/components/NxTag/NxTag.scss

```diff
@@ -39,7 +39,7 @@
     border-color: var(--nx-color-interactive-border-hover);
   }
 
-  &:focus {
+  &:focus-within {
```

Comment:
Why this change? The effect that I would have expected from this change would be that the focus border only shows up due to keyboard nav and not mouse clicks.  However that's not what I'm seeing, the focus still also shows up on mouse clicks just like if we were still using `:focus` here.

Reply:
> Why this change? The effect that I would have expected from this change would be that the focus border only shows up due to keyboard nav and not mouse clicks. However that's not what I'm seeing, the focus still also shows up on mouse clicks just like if we were still using `:focus` here.

The focus styling is applied to the surrounding parent element but it's the `input` that receives the focus so we need to use `:focus-within`

Reply:
> The focus styling is applied to the surrounding parent element but it's the input that receives the focus so we need to use :focus-within

Oh of course, I was thinking of that other new focus thing, `:focus-visible` was it?